src/signals/incident-management/components/Label/__test__/Label.test.js

```diff
@@ -68,7 +68,7 @@ describe.skip('signals/incident-management/components/Label', () => {
     rerender(
       withAppContext(
         <div>
-          <Label htmlFor="someOtherElementId" isGroupHeader={false}>This is my label text</Label>
+          <Label htmlFor="someOtherElementId" isGroupHeader>This is my label text</Label>
```

Comment:
Shouldn't `isGroupHeader` be left out? The value before was `false` and since the component's default value is not `false` as well, you can omit it.

Reply:
gedaan